shared/src/business/useCases/serveCaseToIrs/generateDraftDocument.js

```diff
@@ -0,0 +1,35 @@
+const {
+  replaceBracketed,
+} = require('../../../business/utilities/replaceBracketed');
+
+/**
+ * @param {object} providers the providers object containing applicationContext, caseEntity, document, replacements
+ * @param providers.applicationContext api applicationContext
```

Comment:
I should've caught this on the first PR, but these params all need types

Reply:
Done!